javascript/operators/optional_chaining.json

```diff
@@ -42,18 +52,23 @@
             "nodejs": {
               "version_added": false
             },
-            "opera": {
-              "version_added": "65",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "Experimental JavaScript",
-                  "value_to_set": "true"
-                }
-              ]
-            },
+            "opera": [
+              {
+                "version_added": "67"
+              },
+              {
+                "version_added": "65",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "Experimental JavaScript",
+                    "value_to_set": "true"
+                  }
+                ]
+              }
+            ],
             "opera_android": {
-              "version_added": false
+              "version_added": "67"
```

Comment:
This should actually be “false”; there’s no Opera Android release correlating with Chrome 80.